### DISCUSSION TITLE
new release for LOGBACK-1591

### DISCUSSION THREAD

**Discussion by raphaelauv** [tchiotludo/akhq#953] [General]
Do you plan to release a 0.20 since you merged https://github.com/tchiotludo/akhq/commit/4c01cfdc5df6d8fdd3450718de29e8e120cb7bb5
?
Thank you

**Comment by tchiotludo** [comment]
Next week I think.
There is a major upgrade to micronaut https://github.com/tchiotludo/akhq/pull/945 in the dev branch, I need some feedback if there is no regression

**Reply by MeijerM1** [reply]
Hi,
Since this is a security patch concerning a severe CVE a quicker release would be appreciated.

**Comment by tchiotludo** [comment]
Release is done [here](https://github.com/tchiotludo/akhq/releases/tag/0.20.0) with also the second CVE for logback :smile:
